src/CodeStyle/CSharp/CodeFixes/CSharpFormattingCodeFixProvider.cs

```diff
@@ -13,6 +14,8 @@ internal class CSharpFormattingCodeFixProvider : AbstractFormattingCodeFixProvid
     {
         private readonly EditorConfigOptionsApplier _editorConfigOptionsApplier = new EditorConfigOptionsApplier();
 
+        protected override ISyntaxFormattingService SyntaxFormattingService => new CSharpSyntaxFormattingService();
```

Comment:
is MEF still used in this layer? how common layer do inverse injection for dependency?

Reply:
There is no MEF in this layer.